doc/api/diagnostics_channel.md

```diff
@@ -1117,6 +1117,64 @@ Emitted when server receives a request.
 
 Emitted when server sends a response.
 
+#### Modules
+
+`module.require.start`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
+  * `parentFilename` - Name of the module that attempted to require(id).
+
+Emitted when `require()` is executed
+
+`module.require.end`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
+  * `parentFilename` - Name of the module that attempted to require(id).
+
+Emitted when a `require()` call returns
+
+`module.require.error`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
+  * `parentFilename` - Name of the module that attempted to require(id).
+* `error` {Error}
+
+Emitted when a `require()` throws an error
+
+`module.import.asyncStart`
```

Comment:
Why the `async` prefix here and in `asyncEnd` but not an `asyncError`?
Is there any benefit in adding this prefix? Do we expect a sync variant?

Reply:
This is created by default when a `tracingChannel` is used. It creates a:

* `start`
* `end`
* `asyncStart`
* `asyncEnd`
* `error`

Reply:
When calling passing a promise, it will trigger `async*` automatically. So, it's not a name I've chosen to use. It's just the tracing channel pattern.

Reply:
ah right, sorry.